addons/stock/models/product.py

```diff
@@ -97,6 +97,17 @@ class Product(models.Model):
         compute='_compute_nbr_reordering_rules', compute_sudo=False)
     putaway_rule_ids = fields.One2many('stock.putaway.rule', 'product_id', 'Putaway Rules')
     storage_category_capacity_ids = fields.One2many('stock.storage.category.capacity', 'product_id', 'Storage Category Capacity')
+    show_on_hand_qty_status_button = fields.Boolean(compute='_compute_show_qty_status_button')
```

Comment:
Is there a configuration where the two buttons have different values ? If not I would only use one

Reply:
yep, in enterprise https://github.com/odoo/enterprise/pull/20440/commits/0bda231d6064b3e28e55c4372a600e6a3dc2771c